src/lib/constants.ts

```diff
@@ -52,3 +52,9 @@ export const proposalQuickFilters = ['Grant', 'Activation', 'Wave 4', 'Wave 5']
 export const MAX_NAME_LENGTH_FOR_PROPOSAL = 100
 export const TALLY_DESCRIPTION_SEPARATOR = '  ' // Tally uses double spaces to separate name and description
 export const ipfsGateway = 'https://red-legislative-meadowlark-461.mypinata.cloud/ipfs/'
+
+export const CURRENCY_ADDRESS_SYMBOLS_PAIR = {
```

Comment:
Please move this object to `lib/contracts.ts`. Consider using an existing object `tokenContracts`.

Reply:
Done, used the `tokenContracts` object so as not to introduce a new constant